om4labs/diags/seaice/regrid.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import scipy
+import xarray as xr
+import xesmf as xe
+
+
+def add_matrix_NaNs(regridder):
```

Comment:
should we put that in om4common if we need it in other scripts?

Reply:
Created a broader issue about this - see #12